### DISCUSSION TITLE
GitHub 仓库协管交流群

### DISCUSSION THREAD

**Discussion by Pigeon0v0** [Meloong-Git/PCL#1946] [讨论]
为方便各位 Collaborator 同步进度和部分解答，现本人建立了 QQ 群用于讨论...

QQ 群号：379026872

加群的时候填写一下 GitHub ID...

如果是比较活跃解决问题的，即使未拥有 Collaborator 权限也可以进（

注：目前群管皆为学生，进群审核可能需要一段时间，耐心等待。

**Comment by LTCatt** [comment]
umm，之前有人问过，我 **不建议** 建立私下的 Collaborator 交流群。
Collaborator 之间并不认识，QQ 群这种即时交流可能带来额外的矛盾，因为 Collaborator 的选择是只在技术上做出的决定，并没有考虑一个人的方方面面。
其次，这种闭塞的小团体可能会诱发偏见，一部分人可能盲从他人，导致整个团体都导出了一个不一定正确的 “统一答案”（某种意义上，饭圈就是这样形成的）。对于 GitHub 这样一个开放的、允许思维冲突碰撞的技术交流环境来说，一个“统一答案”不一定算是一件好事。
我可以建立一个 Collaborator 间沟通信息的公开帖子，但我确实不建议建立即时交流、对外不公开的群聊。

**Reply by Pigeon0v0** [reply]
出于方便讨论的目的还是开了个群，有些问题（如游戏崩溃和部分启动器 BUG ）可能需要 IM 性质的软件协助讨论（

**Comment by Deep-Dark-Forest** [comment]
现在这个群与 [PCL Community ](https://github.com/PCL-Community) 交流群合并了，但是讨论的内容基本不变（

**Reply by Deep-Dark-Forest** [reply]
> 艹原来这个群号一直都是 [PCL Community](https://github.com/PCL-Community) 的吗 现在才发现（

额，并不是（
这个群以前是 PCL 仓库协管交流群，后来改名叫了 PCL2 Community
4月份的时候 WFB 建了一个 PCL Community 交流群，不过后来由于这两个群的成员和聊的内容基本相同，WFB 就解散了他建的群，并且原来的 PCL2 Community 改名为 PCL Community，头像也改成了现在 PCL Community 的头像（
